src/NUnitTestExtractor/NUnitTestExtractorTests/TestExtractorTests.cs

```diff
@@ -0,0 +1,255 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+using NUnit.Framework;
+using NUnitTestExtractor;
+using System.Diagnostics;
+using System.Collections;
+using System.Text.RegularExpressions;
+using System.Globalization;
+
+namespace NUnitTestExtractorTests
+{
+    [TestFixture]
+    public class TestExtractorTests
+    {
+
+        /// <summary>
+        /// Used to invoke the GetTests() method in NUnitTestExtractorApp
+        /// </summary>
+        /// <param name="dlls">The dlls to pass to GetTests()</param> 
+        /// <param name="level">The level to pass to GetTests()</param>
+        private static void InvokeGetTestsUsingFileOutput(StringBuilder builder, List<string> dlls, NUnitTestExtractorApp.Level level)
```

Comment:
Since this function isn't using file output anymore, you should rename it to be less confusing.